recipes/pygnome/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "PyGnome" %}
+{% set version = "1.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-ORR-ERD/PyGnome/archive/v{{ version }}.tar.gz
+  sha256: f4263ea7e373414ea66ca0f630b705b07e0131258b2fafad03d369b0f5ef0ee6
+
+build:
+  skip: True  # [py<38]
+  missing_dso_whitelist:
+    - libnetcdf.*.dylib
+    - libmfhdf.*.dylib
+    - libdf.*.dylib
+    - libhdf5_hl.*.dylib
+    - libhdf5.*.dylib
+    - libcurl.*.dylib
+    - libsz.*.dylib
+    - libzip.*.dylib
+    - libblosc.*.dylib
+    - libzstd.*.dylib
+    - libxml2.*.dylib
+  number: 0
+  script: cd py_gnome && {{ PYTHON }} -m pip install ./ -vv
+
+requirements:
+  build:
+    - cython
+    - numpy
+    - netCDF4
+    - scikit-build-core
+    - clang 15.*
+    - cmake
+  host:
```

Comment:
and I think this is the problem -- all those libs it can't find should be in the host requirements, so they'll be there.